fsevents.js

```diff
@@ -6,7 +6,11 @@
 /* jshint node:true */
 'use strict';
 
-var Native = require('./build/Release/fse');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
+var Native = require(binding_path);
```

Comment:
Actually what `require` does isn't relevant, the concern is `binary.find`. I see now this is directly following the [example provided by node-pre-gyp](https://github.com/mapbox/node-pre-gyp#3-dynamically-require-your-node). Still seems like overkill.